### PR TITLE
Working towards Django 2 compatibility

### PR DESCRIPTION
**Pull request by dannyroberts**
This isn't comprehensive, but it's the lower-impact changes that came up. Thought I'd get them in the pipeline

### CODE REVIEW

File: requirements/requirements.in [hunk 1]
```diff
@@ -65,7 +65,7 @@ django-countries==4.6
 reportlab==3.0
 stripe==1.67.0
 django-compressor==2.1
-django-angular==0.7.16
+django-angular>=0.7.16
```
Comment (emord) [inline requirements/requirements.in]: Is this accurate? Can you import from both `djangular` and `djng` in 0.7.16?

**Review by emord — COMMENTED** [review]
I think that you also need to add the django-angular upgrade to the python2 requirements unless you aren't merging this for a week

File: requirements/requirements.in (hunk 1 above)
Comment (dannyroberts) [reply requirements/requirements.in]: nope, you're right. Lemme figure this out and I'll update

**Comment by snopoke** [comment]
What's the plan for testing this? I'm particularly concerned with changes to Angular which could affect the ICDS dashboard.

**Comment by dannyroberts** [comment]
Understood. I'm going to be pretty careful here and I'll make sure to check
back in with you specifically about the dashboard if we do end up trying to
push forward with the angular change. I may also split out the smaller
things and merging them separately.

**Comment by dannyroberts** [comment]
Pulled out what's usable in this PR to a new one: https://github.com/dimagi/commcare-hq/pull/26778. Omitting the angular change for now

**Pull request closed without merging**